src/main/java/reposense/commits/CommitInfoAnalyzer.java

```diff
@@ -23,21 +23,25 @@
  * Analyzes commit information found in the git log.
  */
 public class CommitInfoAnalyzer {
+    public static final DateFormat GIT_STRICT_ISO_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+
     private static final Logger logger = LogsManager.getLogger(CommitInfoAnalyzer.class);
     private static final String MESSAGE_START_ANALYZING_COMMIT_INFO = "Analyzing commits info for %s (%s)...";
 
-    private static final DateFormat GIT_ISO_FORMAT = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
-    private static final String LOG_SPLITTER = "\\|";
+    private static final String LOG_SPLITTER = "\\|\\n\\|";
 
     private static final int COMMIT_HASH_INDEX = 0;
     private static final int AUTHOR_INDEX = 1;
     private static final int EMAIL_INDEX = 2;
     private static final int DATE_INDEX = 3;
-    private static final int MESSAGE_INDEX = 4;
+    private static final int MESSAGE_TITLE_INDEX = 4;
+    private static final int MESSAGE_BODY_INDEX = 5;
 
     private static final Pattern INSERTION_PATTERN = Pattern.compile("([0-9]+) insertion");
     private static final Pattern DELETION_PATTERN = Pattern.compile("([0-9]+) deletion");
 
+    private static final Pattern LEADING_4_SPACES_PATTERN = Pattern.compile("^ {4}", Pattern.MULTILINE);
```

Comment:
call it `MESSAGEBODY_LEADING_PATTERN` instead, saying its 4 spaces doesn't tell much